nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-controller-services.js

```diff
@@ -497,6 +497,36 @@ nf.ControllerServices = (function () {
     };
 
     /**
+     * Determines if the user is authorized for the parent of the specified controller service.
+     *
+     * @param dataContext
+     * @returns {boolean}
+     */
+    var isAuthorizedForControllerServiceParent = function (dataContext) {
```

Comment:
The method naming seems imprecise here, how about `canWriteControllerServiceParent` or something that clarifies which permission or action is authorized?